.gitignore

```diff
@@ -104,4 +104,6 @@ db.env
 space2stats_api/space2stats_env
 *.env
 cdk.out
-lambda_layer
\ No newline at end of file
+lambda_layer
+
+.benchmarks/
```

Comment:
Out of curiosity, what's in the benchmarks?

Reply:
it's when you have `pytest-benchmark` installed 

let me add this to my .gitignore global instead